### PR TITLE
Add the "scriptElementAdded" event (closes #996)

### LINKED ISSUE #996: Raise an event when a new script was added to a page
It's necessary for https://github.com/DevExpress/testcafe/issues/1072

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @churkin @miherlosev

### CODE REVIEW

File: test/client/fixtures/sandbox/node/element-test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+test('Check the "scriptElementEvent" event is raised', function () {
+    window.script = document.createElement('script');
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/element-test.js]: lowercase test name
don't save to window, just use `var`

File: test/client/fixtures/sandbox/node/element-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    function handler (e) {
+        strictEqual(e.el, window.script);
+
+        hammerhead.off(hammerhead.EVENTS.scriptElementAdded, handler);
+    }
+
+    hammerhead.on(hammerhead.EVENTS.scriptElementAdded, handler);
+
+    eval(processScript('document.body.appendChild(window.script)'));
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/element-test.js]: We don't use code instrumentation for methods.
Just write `document.body.appendChild(script)`

File: src/client/sandbox/node/element.js [hunk 3]
```diff
@@ -34,7 +34,8 @@ export default class ElementSandbox extends SandboxBase {
 
         this.overridedMethods = null;
 
-        this.BEFORE_FORM_SUBMIT = 'hammerhead|event|before-form-submit';
+        this.BEFORE_FORM_SUBMIT   = 'hammerhead|event|before-form-submit';
+        this.SCRIPT_ELEMENT_ADDED = 'hammerhead|event|script-added-event';
```
Comment (churkin) [inline src/client/sandbox/node/element.js]: `'hammerhead|event|script-added-event'` -> `'hammerhead|event|script-added'`

**Comment by AlexanderMoskovkin** [comment]
FPR

**Pull request merged**